openapi/openapi_spec_resource_schema_definition_property_test.go

```diff
@@ -324,7 +560,7 @@ func TestTerraformSchema(t *testing.T) {
 			ReadOnly: false,
 			Required: true,
 		}
-		Convey("When createTerraformPropertyBasicSchema method is called", func() {
+		Convey("When terraformSchema method is called", func() {
 			tfPropSchema, err := s.terraformSchema()
 			Convey("Then the resulted terraform property schema should be of type int too", func() {
```

Comment:
Again feels like this could be written as a loop

Reply:
as discussed, tackling this in a separate changeset